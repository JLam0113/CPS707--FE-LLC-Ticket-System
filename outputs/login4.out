Welcome, please enter a command.
Please enter your username
