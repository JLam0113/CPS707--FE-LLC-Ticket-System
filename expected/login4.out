Welcome, please enter a command.
Please enter your username
Non-privileged Login successful, now logged in as user1, please enter a command
Invalid command (cannot login while logged in), please enter a command
Session ended
